DEVELOPING_OPAL/validate/src/test/java/org/opalj/fpcf/fixtures/string/tools/GreetingService.java

```diff
@@ -0,0 +1,9 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.fpcf.fixtures.string.tools;
+
+public interface GreetingService {
```

Comment:
Should we give the classes and methods in this hierarchy more descriptive names? In the context of these tests, it doesn't matter that they produce greetings, it is about what they do with strings.